go/vt/vtgate/planbuilder/operators/update.go

```diff
@@ -150,66 +166,104 @@ func createUpdateOperator(ctx *plancontext.PlanningContext, updStmt *sqlparser.U
 		}
 	}
 
-	vp, cvv, ovq, subQueriesArgOnChangedVindex := getUpdateVindexInformation(ctx, updStmt, vindexTable, qt.ID, assignments)
+	target := updStmt.TableExprs[0]
+	atbl, ok := target.(*sqlparser.AliasedTableExpr)
```

Comment:
shouldn't we not also check the size of `updStmt.TableExprs`? You can have multiple AliasedTableExpr next to each other by just coma separating the table names, I belive.

Reply:
currently, it is unsupported by the semantic analyzer.